x/data/server/features/anchor.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Anchor
+
+  Scenario: data is anchored when the data has not been anchored
+    Given a valid content hash
+    When alice attempts to anchor the data at block time "2020-01-01"
+    Then an error of ""
```

Comment:
```suggestion
```

we can just skip checking for an error because we don't expect one

Reply:
duplicate